### ISSUE TITLE
Build error in alpha-0.8: No rule to make target '/usr/lib/x86_64-linux-gnu/libGLU.so', needed by 'src/renderergl1/librenderergl1.so'.  Stop.

### ISSUE THREAD

**Issue by cengique** [GrangerHub/tremulous#32]
```
$ make 
...
[ 63%] Building C object src/renderergl1/CMakeFiles/renderergl1.dir/__/qcommon/q_math.c.o
make[3]: *** No rule to make target '/usr/lib/x86_64-linux-gnu/libGLU.so', needed by 'src/renderergl1/librenderergl1.so'.  Stop.
make[2]: *** [CMakeFiles/Makefile2:655: src/renderergl1/CMakeFiles/renderergl1.dir/all] Error 2
make[1]: *** [CMakeFiles/Makefile2:844: src/client/CMakeFiles/tremulous.dir/rule] Error 2
make: *** [Makefile:287: tremulous] Error 2
```
But I do have `libGLU.so` under `/usr/lib/` and '/usr/lib64/'. Not sure why the makefile is looking under `/usr/lib/x86_64-linux-gnu/`

**Comment by wtfbbqhax** [comment]
Does the Makefile work?

**Comment by cengique** [comment]
No, it fails. I looked inside `src/client/CMakeFiles/tremulous.dir/build.make` and there are several references to the `/usr/lib/x86_64-linux-gnu` folder. So I ended up making a symbolic link and the Makefile works now. This is on a Gentoo 64-bit system.

**Comment by wtfbbqhax** [comment]
I mean the top level Makefile- not the cmake generated file.. 

That said, I'm guessing that you're running the `cmake` at the top level and clobbering the stock makefile.

**Comment by cengique** [comment]
Yeah, that's what I did. I ran 
```
cmake .
```

And then 
```
make
```

Should I revert to the `Makefile` from your repo?

**Comment by wtfbbqhax** [comment]
Yes, currently the CMake is lacking behind in functionality, such as packing up the default qvm's and assets, setting the release version in the binary et. al. 

This lack of functionality is largely due to not having figured out how to build Windows executables from the cmake project- until then, I'm just keeping it functional enough to not be blatantly broken. 

Regarding your build failure, I can only suspect that the issue is with the system provided `FindOpenGL.cmake` (Included in `src/renderergl1/CMakeFiles.txt:3`), not sure how to fix that at this time.

**Comment by wtfbbqhax** [comment]
Just a note, this is how I do Cmake builds

```
cd tremulous/
mkdir alt
cd alt/
cmake ..
make
```

This makes it easier to cleanup the old artifacts later

**Comment by cengique** [comment]
Oh, yeah, the regular `make` worked! For some reason I thought the README instructions were old and I was supposed to use `cmake`. 
Thanks for the cmake recipe, it's a bit messy in my repo because of this exact reason.
